README.md

```diff
@@ -242,18 +242,61 @@ Also missing are the following.
 
 ## Installing the reference implementation
 
+Currently, there are two ways of installing the reference implementation.
+You may install it on an existing Raspbian system or create a new image.
+The two alternatives are presented below.
+
+### Install on an existing system
+
 You'll need to install the required libraries and project.
 Here is how you can do it under the Ubuntu GNU/Linux distribution.
 
 ```bash
 sudo apt-get install git libbluetooth-dev libglib2.0-dev python3-dev python3-setuptools shellcheck sqlite3 virtualenv dh-virtualenv debhelper supervisor
-git clone https://github.com/dspinellis/epidose
+git clone https://github.com/eellak/epidose
 cd epidose
 virtualenv venv -p /usr/bin/python3
 . venv/bin/activate
 pip3 install -e .
 ```
 
+### Create a new image
+
+You'll need to download a vanilla Raspbian distribution (Lite version is suggested for this purpose)
+and burn it on a MicroSD card.
+Note that you must enable the SSH daemon on the corresponding image.
+Then execute the following steps:
+
+* In the /etc/ansible/ansible.cfg file uncomment and set as True the following:
+```bash
+allow_world_readable_tmpfiles = True
+pipelining = True
+```
+
+* In the /etc/ansible/hosts file create the group of usernames and  IP addresses for each Raspberry device, e.g.,
+```bash
+   [epidose]
+   192.168.0.105 ansible_user=pi ansible_connection=ssh ansible_private_key_file=~/ssh/id_rsa
```

Comment:
How is the key provided?

Reply:
The private key there can be anything that the author wants.
So if we have the private/public key in our systems we can execute the ansible script
and that will use the ssh-copy-id to any instances found under the [epidose] group.

Reply:
You need to provide details and instructions.

Reply:
If there is an appropriate explanation online, link to it.